docs/providers/documentation/keep-provider.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: "Keep"
+sidebarTitle: "Keep Provider"
+description: "Keep provider acts as a placeholder for alerts without a specific provider."
```

Comment:
It actually allows you to query the alerts you have in Keep, can we adjust that?

Reply:
Will add this also